bool enableStreaming
bool blockUntilStarted
int32 numFramesToStream
---
